### PR TITLE
Story/datause widget

### PR DESCRIPTION
**Pull request by kandaj**

### CODE REVIEW

File: public/locales/en-GB/translation.json [hunk 1]
```diff
@@ -395,7 +395,7 @@
                 "cancel": "Cancel"
             },
             "howToHeader": "How to start using your widget",
-            "howToDesc": "Below is an interactive example of how your widget could look and be used. Please note that this design is compatible with mobile devices and can be resized to fit anywhere on your website. To start using this widget please accept the terms and conditions by clicking on the 'Get widget' button below.",
```
Comment (ThinkingMonkeyLtd) [inline public/locales/en-GB/translation.json:398]: Conditions(s)?

File: src/components/Cta/Cta.stories.mdx [hunk 2]
```diff
@@ -4,14 +4,22 @@ import closeIcon from '../../images/close-alt.svg';
 import Icon from '../Icon';
 import Cta from './Cta';
 
+<<<<<<< HEAD
```
Comment (ThinkingMonkeyLtd) [inline src/components/Cta/Cta.stories.mdx:7]: Conmflicts

File: src/components/Cta/__snapshots__/Cta.test.js.snap [hunk 3]
```diff
@@ -3,10 +3,17 @@
 exports[`Given the Alert component When it is rendered Then matches the previous snapshot 1`] = `
 <div>
   <a
+<<<<<<< HEAD
```
Comment (ThinkingMonkeyLtd) [inline src/components/Cta/__snapshots__/Cta.test.js.snap:6]: Conflicts

File: src/pages/commonComponents/AdvancedSearchModal/AdvancedSearchModal.js [hunk 4]
```diff
@@ -0,0 +1,202 @@
+import React, { useState } from 'react';
```
Comment (ThinkingMonkeyLtd) [inline src/pages/commonComponents/AdvancedSearchModal/AdvancedSearchModal.js:1]: Don't think this exists in dev / release anymore

File: src/pages/dashboard/Components/ActivityLogCard/__snapshots__/ActivityLogCard.test.jsx.snap [hunk 5]
```diff
@@ -52,7 +52,11 @@ exports[`Given the ActivityLogCard component When it is rendered Then matches th
               8 November 2021
             </h3>
             <div
+<<<<<<< HEAD
```
Comment (ThinkingMonkeyLtd) [inline src/pages/dashboard/Components/ActivityLogCard/__snapshots__/ActivityLogCard.test.jsx.snap:55]: Conflicts

**Review by ThinkingMonkeyLtd — CHANGES_REQUESTED** [review]

**Review by kymmeh — APPROVED** [review]

**Review by kymmeh — APPROVED** [review]

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Review by ThinkingMonkeyLtd — APPROVED** [review]

**Pull request merged**